src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -55,21 +55,28 @@ namespace System.Buffers
 }
 namespace System.Numerics.Tensors
 {
-    [System.Diagnostics.CodeAnalysis.Experimental("SYSLIB5001", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
-    public partial interface IReadOnlyTensor<TSelf, T> : System.Collections.Generic.IEnumerable<T>, System.Collections.IEnumerable where TSelf : System.Numerics.Tensors.IReadOnlyTensor<TSelf, T>
+    [System.Diagnostics.CodeAnalysis.ExperimentalAttribute("SYSLIB5001", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Oh boy, this file is full of edge cases I should add to ApiDiff.

Reply:
Thats good! Glad this mess of a file can help.